library/alloc/src/lib.rs

```diff
@@ -85,6 +85,7 @@
 //
 // Library features:
 // tidy-alphabetical-start
+#![cfg_attr(not(no_global_oom_handling), feature(string_replace_in_place))]
```

Comment:
Fwiw I think you may as well drop the `cfg_attr` and enable the feature unconditionally, usually we don't gate these too strictly. And we'll wind up using it elsewhere anyway.

I'll merge after PR CI finishes, with or without this (just ping me once it does if I miss it)

Reply:
Yeah; I'm sorry, this was bad advice.

Reply:
I undid this (that is, undid removing `cfg_attr`, so re-added it) and re-pushed

@rustbot ready